roles/security_errata/tasks/main.yml

```diff
@@ -18,6 +18,7 @@
 
 - include: CVE-2015-0235.yml
   tags: CVE-2015-0235
+  when: ansible_architecture != "ppc64le"
```

Comment:
Binary provided at ```ursula/roles/security_errata/files/detect_CVE-2015-0235``` gives ```Exec format error``` upon its execution. Hence we skipped it temporarily. Supporting binary for ppc64le could be updated later.